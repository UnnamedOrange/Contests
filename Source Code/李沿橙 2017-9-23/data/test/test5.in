5
49
42 468 335 501 170 725 479 359 963 465 706 146 282 828 962 492 996 943 828 437 392 605 903 154 293 383 422 717 719 896 448 727 772 539 870 1000 668 300 36 895 704 812 323 334 674 665 142 712 254 
4
1 1 1 2 
21
1 1 1 1 3 3 3 4 4 5 5 5 5 6 6 6 6 6 7 8 9 
9
1 100 101 102 103 104 105 106 107 
7
4 16 5 7 108 2 17 
