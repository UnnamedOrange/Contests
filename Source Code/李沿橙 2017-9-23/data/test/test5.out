36 42 142 146 170 282 293 300 323 334 335 383 392 422 437 448 465 468 492 501 539 665 668 674 704 706 712 717 719 725 772 812 828 870 895 896 903 962 963 996 1000 943 828 727 605 479 359 254 154 
1 1 1 2 
1 1 1 1 3 3 4 4 5 5 5 6 6 6 6 6 8 9 7 5 3 
1 100 102 103 104 105 106 107 101 
2 4 5 7 16 108 17 
